src/racer/util.rs

```diff
@@ -23,6 +27,114 @@ pub fn is_ident_char(c: char) -> bool {
     c.is_alphanumeric() || (c == '_') || (c == '!')
 }
 
+// to ignore C-Style Comment
+#[derive(Debug)]
+enum CommentAutomatonState {
+    None,
+    Start, // just / occurs
+    InComment, // /* ...
+    BeforeEnd, // /* ... *
+    InSlash, // like ///// (maybe not useless)
+}
+
+impl CommentAutomatonState {
+    // in case we can know wer'e in comment without looking ahead
```

Comment:
I don't understand this comment; or maybe it's that the method name doesn't match the comment?

Typo: Should be "we're"